src/utils/getFilteredTodosByStatus.ts

```diff
@@ -0,0 +1,15 @@
+import { Filter } from '../types/Filters';
+import { Todo } from '../types/Todo';
+
+export const getFilteredTodosByStatus = (todos: Todo[], filter: Filter) => {
+  let preparedTodos = [...todos];
+
+  switch (filter) {
+    case Filter.Active:
+      return (preparedTodos = preparedTodos.filter(todo => !todo.completed));
+    case Filter.Completed:
+      return (preparedTodos = preparedTodos.filter(todo => todo.completed));
+    default:
```

Comment:
```suggestion
  switch (filter) {
    case Filter.Active:
      return todos.filter(todo => !todo.completed));
    case Filter.Completed:
      return todos.filter(todo => todo.completed));
    default:
```
- `filter` doesn't mutate an array, so you don't need a copy
- You don't need to assign to a variable, you can just return values from function